fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ValueFormatExceptionMapper.java

```diff
@@ -40,7 +40,7 @@
 
     @Override
     public Response toResponse(final ValueFormatException e) {
-        LOGGER.debug(
+        LOGGER.error(
                 "ValueFormatException intercepted by ValueFormatExceptionMapper: \n", e);
```

Comment:
Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`